front/components/sparkle/AppLayout.tsx

```diff
@@ -188,7 +188,7 @@ export default function AppLayout({
 
             <div
               className={classNames(
-                "flex h-full w-full flex-col",
```

Comment:
Isn't `height:100%` supposed to be 100% of the viewport? 🤔

Reply:
h-full is 100% , it's 100% of the parent div - which only works if the parent has a fixed size. 100% of the viewport is 100vh.

Reply:
in both cases it's 80 pixel too much :-)

Reply:
Yeah yeah sure. Then I guess to properly we would have to modify the layout to have the banner on an other z-index, right?
Anyway, that solves it thanks 🙏

Reply:
Wait why do we need this, given that we have access to the parent div and could constraint there?

Reply:
It doesn't seem trivial. Looked a bit and the layout seems a bit flaky 🤔